<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LENEP/UENF - CS</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Arquivo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtArquivo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnArquivo">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPercentualSubAmostragem">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>65</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>65</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Percentual de subamostragem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProcessar">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>65</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Processar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>105</y>
     <width>671</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Informações do Dado</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>nt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>nr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxtNt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>45</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxtNr">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>45</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamanho iava</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxtIava">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>45</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnVisualizarDadoOriginal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualizar Dado Original</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVisualizarDadoObservado">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>265</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualizar Dado Observado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInverterFft">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inverter com FFT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVisualizarDadoRecuperado">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>265</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualizar Dado Recuperado</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
